charms/kfp-api/src/charm.py

```diff
@@ -180,11 +180,13 @@ def service_environment(self):
     @property
     def _kfp_api_layer(self) -> Layer:
         """Create and return Pebble framework layer."""
+        # The service name should be the same as the one
+        # defined in the Rockcraft project: apiserver
         layer_config = {
             "summary": "kfp-api layer",
             "description": "Pebble config layer for kfp-api",
             "services": {
-                self._container_name: {
```

Comment:
why do we change it in only one place?

Reply:
What do you mean only in one place?

Reply:
you changed attributed to hardcoded value. I am curious if that should be done in other places as well